misaligned.cpp

```diff
@@ -1,21 +1,52 @@
-#include <iostream>
-#include <assert.h>
+#include "misaligned.h"
 
-int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
-    for(i = 0; i < 5; i++) {
-        for(j = 0; j < 5; j++) {
-            std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
+namespace misaligned
+{
+
+    const char *majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+    int numberOfMajorColors = sizeof(majorColor) / sizeof(majorColor[0]);
+    const char *minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+    int numberOfMinorColors = sizeof(minorColor) / sizeof(minorColor[0]);
+
+    int Misaligned::printColorMap()
+    {
+        int i = 0, j = 0;
+        for (i = 0; i < 5; i++)
+        {
+            for (j = 0; j < 5; j++)
+            {
+                std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";

```

Comment:
The number 5 would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work if major color or minor color length is other than 5. Please think over it and write test cases accordingly to cover such cases.